target/linux/ramips/dts/mt7628an_joowin_jw-wr758ac.dtsi

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "joowin,jw-wr758ac", "mediatek,mt7628an-soc";
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi-high {
+			label = "blue:wifi-high";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi-med {
+			label = "blue:wifi-med";
+			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi-low {
+			label = "blue:wifi-low";
+			gpios = <&gpio 37 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&state_default {
+	gpio {
+		groups = "i2c", "wled_an", "gpio", "refclk", "wdt", "p0led_an";
```

Comment:
Do you need i2c and p0led_an? It seems this would be enough:
```diff
 &state_default {
 	gpio {
-		groups = "i2c", "wled_an", "gpio", "refclk", "wdt", "p0led_an";
+		groups = "refclk", "wdt", "wled_an";
 		function = "gpio";
 	};
 };
```

Reply:
Thanks, fixed in latest push.